### DISCUSSION TITLE
Grove - I2C Motor Driver (L298P) | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#127] [Q&A]
# Grove - I2C Motor Driver (L298P) | Seeed Studio Wiki

Grove - I2C Motor Driver(L298P) is a common-use motor driver for stepping motor and servo motor. It embeds an STM32 chip for burning the code to control the

https://wiki.seeedstudio.com/Grove-I2C-Motor-Driver-L298P/

**Comment by besbar** [comment]
Hello ! When using a stepper motor, does the library permit speed motor control ?

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.

**Comment by mint-n-pepper** [comment]
I was wondering: Can I use this motor driver with a 3.3V board such as the microbit when the jumper for external power supply is removed?
